### DISCUSSION TITLE
Sequent Microsystems - Raspberry Pi Pool Controller

### DISCUSSION THREAD

**Discussion by expresbodhi** [tagyoureit/nodejs-poolController#856] [General]
I just got the following Email from Sequent Microsystems, Seems like a good opportunity for the community to provide feedback.
<img width="569" alt="image" src="https://github.com/tagyoureit/nodejs-poolController/assets/133036749/1662344e-16cb-4d09-ac6c-8107628fa6ea">

**Comment by gw8674** [comment]
Yes. And here is the link.
https://forum.sequentmicrosystems.com/index.php?threads/pool-automation-controller.31/
